docs/troubleshooting.md

```diff
@@ -290,6 +290,19 @@ There was an issue with the installing of the drivers certificate, so the driver
 [This Guide](guests.md#upgrade-from-citrix-xenserver-client-tools) may help you through the process.
 
 
+### My Windows VM can't connect to a secure internet after installing PV-tools/drivers (E.G SSL errors when trying to access websites, copying files from a pc to the VM ETC.)
+
+#### Cause
+TCP offloading is enabled for Citrix PV drivers in windows, This can cause TSL/SSL problems.
```

Comment:
1. I think TCP offloading is enabled in all PV drivers, not specifically Citrix PV drivers, so maybe no need to put Citrix in there
2. I would say "[…] in Windows, that could cause TLS/SSD problems" (see the multiple typos and wrong/missed caps)